### PR TITLE
[HIPIFY][doc] ToC fix

### PR DESCRIPTION
**Pull request by LisaDelaney**
Minor aesthetic changes and a ToC fix.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,4 +53,4 @@ hipify-clang --md --doc-format=full --doc-roc=separate
 To generate this documentation in CSV, use the `--csv` option instead of `--md`. Instead of using
 the `full` format, you can also build in `strict` or `compact` format.
 
-To see all available options use the `--help` or `--help-hidden` HIPIFY-Clang option.
+To see all available options, use the `--help` or `--help-hidden` HIPIFY-Clang option.
```
Comment (emankov) [inline README.md]: `HIPIFY-Clang` -> `hipify-clang`

**Review by emankov — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (LisaDelaney) [reply README.md]: Should that text be in code quotes then?

Comment (emankov) [reply README.md]: The same style, I think: `hipify-clang`

**Review by emankov — APPROVED** [review]
LGTM, thanks!

**Pull request merged**